pkg/users/local.js

```diff
@@ -373,7 +373,7 @@ PageAccountsCreate.prototype = {
         var self = this;
         $('#accounts-create-cancel').on('click', $.proxy(this, "cancel"));
         $('#accounts-create-create').on('click', $.proxy(this, "create"));
-        $('#accounts-create-dialog .check-passwords').on('keydown', $.proxy(this, "validate"));
+        $('#accounts-create-dialog .check-passwords').on('keydown change', $.proxy(this, "validate"));
```

Comment:
ah, I forgot that you've mentioned two places for this. In selenium it happens only sometimes, so I did not catch that.